appservice/src/createAppService/SiteCreateStep.ts

```diff
@@ -32,11 +39,20 @@ export class SiteCreateStep extends AzureWizardExecuteStep<IAppServiceWizardCont
                     serverFarmId: wizardContext.plan ? wizardContext.plan.id : undefined,
                     clientAffinityEnabled: wizardContext.newSiteKind === AppKind.app,
                     siteConfig: await this.getNewSiteConfig(wizardContext)
-                    });
-                const createdNewApp : string = localize('CreatedNewApp', 'Created new {0} "{1}": {2}', getAppKindDisplayName(wizardContext.newSiteKind), wizardContext.site.name, `https://${wizardContext.site.defaultHostName}`);
+                });
+                const createdNewApp: string = localize('CreatedNewApp', 'Created new {0} "{1}": {2}', getAppKindDisplayName(wizardContext.newSiteKind), wizardContext.site.name, `https://${wizardContext.site.defaultHostName}`);
                 ext.outputChannel.appendLine(createdNewApp);
                 ext.outputChannel.appendLine('');
-                window.showInformationMessage(createdNewApp);
+                const viewLogs: MessageItem = {
+                    title: localize('viewLogs', 'View Logs')
+                };
+
+                // Note: intentionally not waiting for the result of this before returning
+                window.showInformationMessage(createdNewApp, viewLogs).then((result: MessageItem | undefined) => {
```

Comment:
Did you want the response in telemetry?

Reply:
It would take some changes in the UI package to pass along telemetry stuff down through the wizard and I don't think it's worth it for this response. Filed an issue to fix later, though: https://github.com/Microsoft/vscode-azuretools/issues/219